.travis.yml

```diff
@@ -1,21 +1,10 @@
 language: node_js
 node_js: 'lts/*'
-cache:
```

Comment:
npm caching is [now automatic](https://docs.travis-ci.com/user/caching/#npm-cache)